service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -160,7 +160,7 @@ private Run tryUpdateRun(Run r) {
       recordOutboundApiRequestCompletion("wes/runSummary", getStatusStartNanos, getStatusSuccess);
     }
 
-    CbasRunStatus newStatus = CbasRunStatus.INITIALIZING;
+    CbasRunStatus newStatus = CbasRunStatus.UNKNOWN;
```

Comment:
Then, when we want to refresh the status of a particular run, it should default to UNKNOWN if we have no info about said workflow yet, rather than INITIALIZING. This agrees with the expected behavior of UNKNOWN representing missing information from Cromwell, rather than INITIALIZING representing "we think it's been started but it could be an actual error". INITIALIZING should then only correspond to "we sent the workflow to be executed" and last until we receive an actual update about the workflow status (even if that status is UNKNOWN).

Reply:
TOL: Should we set the `newStatus` here to be same as the current status of Run that we have in CBAS database? This way if CBAS was unable to fetch updated status from Cromwell, the new status would be same as the previous one from CBAS database. This would be a better user experience. Otherwise there is a chance that we set the status of Run from `Initializing` -> `Running` -> `Unknown` -> terminal state no? Which would be a weird user experience.

Reply:
My thought was that this should represent the status to the best of our current knowledge. I think we should set it to UNKNOWN if we don't know anything about the current status as I don't see any reason Cromwell would go from running -> not having any idea about the specific workflow -> terminal state, it should just go from running -> terminal state or running -> not having any idea about the specific workflow (if something goes horribly wrong).

Reply:
Based on the code it seems if we don't receive a response from Cromwell or get an error while fetching the status CBAS would set the status to `Unknown` instead of its current state. I also feel we should not go from a `Running` or `Initializing` state to `Unknown`. But I am interested in hearing from other reviewers. @cjllanwarne what do you suggest?